cmd/snap/cmd_wait_test.go

```diff
@@ -57,3 +58,114 @@ func (s *SnapSuite) TestCmdWait(c *C) {
 	// hands control back to the process
 	c.Check(n > 2, Equals, true)
 }
+
+func (s *SnapSuite) TestCmdWaitMissingConfKey(c *C) {
+	n := 0
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		n++
+	})
+
+	_, err := snap.Parser().ParseArgs([]string{"wait", "snapName"})
```

Comment:
Can you please add tests for the Easter egg as well.

Reply:
I removed it again.